src/NCronJob/Scheduler/JobQueueManager.cs

```diff
@@ -130,4 +115,6 @@ public void Dispose()
 
     private void CallCollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
         => CollectionChanged?.Invoke(sender, e);
+
+    public CancellationTokenSource GetCancellationTokenSource(string queueName) => jobCancellationTokens[queueName];
```

Comment:
nitpicK: I try to put public methods on top (after ctor)

Reply:
Fixed